modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1833,12 +1892,86 @@ private String _getUniqueUrlTitle(BlogsEntry entry, String newTitle) {
 				BlogsEntry.class.getName(), "urlTitle", urlTitle);
 		}
 
-		return _friendlyURLEntryLocalService.getUniqueUrlTitle(
-			entry.getGroupId(),
-			_classNameLocalService.getClassNameId(BlogsEntry.class),
+		Set<Long> groupIds = new HashSet<>();
+
+		long globalGroupId = _getGlobalGroudId();
+
+		if (globalGroupId == entry.getGroupId()) {
+			groupIds.addAll(_getSiteGroupIds(entry.getCompanyId()));
+		}
+		else if (globalGroupId != 0) {
+			groupIds = _getCurrentAndAncestorSiteAndDepotGroupIds(
+				entry.getGroupId());
+
+			groupIds.add(globalGroupId);
+		}
```

Comment:
I don't understand this. Why are you making the url title globally unique? Friendly url should be unique per group, not globally unique.

If you want to disambiguate, maybe the solution would be to make friendly urls (not url titles) composite, like we did for documents and media? When generating the friendly url from the url title of a document, we prefix the group name if it is not the current one (e.g. `/d/foo` -> `/d/my-group/foo`.

Reply:
Hi @adolfopa, I've read all of your comments. Thanks for your feedback. I'd not understood the friendly url and the url title properly. I made the code to be more complex. :))
Currently, I involve in another ticket. So I send this issue back to the queue. The product team can resolve it better than I. I'll follow this :eyes: . 
Please help me to close this PR.
Thanks a lot.
Tai.